src/main/java/frc/robot/subsystems/ball/intake/IntakeImpl.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.subsystems.ball.intake;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import ca.team3161.lib.robot.subsystem.RepeatingPooledSubsystem;
+import java.util.EnumMap;
+import java.util.concurrent.TimeUnit;
+
+public class IntakeImpl extends RepeatingPooledSubsystem implements Intake {
+
+    public enum motorDirections {
```

Comment:
Enums should be named following the same convention as classes, ie this should be `MotorDirections`